### PR TITLE
Add missing (standard track) attributes for HTMLInputElement API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of a project to add missing interfaces and interface features to BCD that are from a spec in Editor's Draft or more and is supported in at least one browser.  This particular PR adds missing features, populating the results using data from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.6), for the HTMLInputElement API.

Spec: https://html.spec.whatwg.org/multipage/input.html#htmlinputelement
IDL: https://github.com/w3c/webref/blob/master/ed/idl/html.idl

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Note: the diff looks really weird on this PR, as if features are removed and/or renamed.  No features are being removed in this PR.

**Comment by foolip** [comment]
@vinyldarkscratch are updates to input.json also needed?

**Comment by vinyldarkscratch** [comment]
Not `input.json`, but most likely the individual input types.  I didn't want to include those updates in this PR since it's already a large diff, and do the updates in a follow-up.  I could include it in this PR if preferred, however?

**Comment by foolip** [comment]
OK, let's leave this issue as just updating `HTMLInputElement` APIs. Can you file an issue about what follow-up syncing might be needed?

File: api/HTMLInputElement.json [hunk 1]
```diff
@@ -543,13 +534,15 @@
...
+              "version_added": "3"
             },
             "firefox_android": {
               "version_added": "4"
             },
             "ie": {
-              "version_added": "5.5"
+              "version_added": "10",
+              "partial_implementation": true,
+              "notes": "This property is read-only."
```
Comment (foolip) [inline api/HTMLInputElement.json:545]: This and other notes aren't being added/removed in this PR, it's just entries that appear to be changed in the diff due to other entries being added. I reviewed the actual differences using https://github.com/mdn/browser-compat-data/pull/6862

**Review by foolip — APPROVED** [review]

**Comment by Elchi3** [comment]
Looks like this PR adds a duplicate `width` entry and breaks the BCD build.

**Comment by vinyldarkscratch** [comment]
I quickly merged #7751 which is a hotfix PR that resolves the duplicate entry!

**Comment by Elchi3** [comment]
Thanks @vinyldarkscratch :+1:

**Comment by foolip** [comment]
@Elchi3 thanks for spotting that! The CI did pass, but when merging a bunch of PRs in a row this kind of thing can happen. Sorry!

**Comment by Elchi3** [comment]
No worries! Both of you rock for getting all this data in :)

**Pull request merged**